wallet-connect/src/client.rs

```diff
@@ -161,9 +161,9 @@ impl Client {
             .request(
                 "personal_sign",
                 vec![
-                    message.to_string(),
+                    format!("0x{}", hex::encode(message)),
                     format!("{:?}", address),
-                    "".to_string(),
+                    // "".to_string(), // TODO is password needed?
```

Comment:
shouldn't it send at least empty string if no password?

Reply:
This aligns with https://github.com/WalletConnect/walletconnect-example-dapp/blob/79e427903f66ca0070ca3551c2391609b56be382/src/App.tsx#L544-L547

Reply:
- [x] Metamask 
- [x] Defi Desktop Wallet
<img width="866" alt="image" src="https://user-images.githubusercontent.com/92778185/199902916-4319220c-f9c3-40c5-8d85-19b39926ba54.png">
- [x] Trust Wallet

Defi Mobile Mobile can not show the string (maybe need to report to them)